draft-ietf-jose-hpke-encrypt.md

```diff
@@ -269,41 +269,51 @@ no additional processing requirements are introduced by HPKE-based key encryptio
+Where:
 
-- recipient_protected_header (object): This member contains the base64url-encoded JWE Per-Recipient Unprotected Header (see JWE JSON Serialization in {{Section 7.1 of RFC7156}} of the recipients member. To serialize this header member the procedure from Section 3.3 of RFC 7638 MUST be used. Unlike with RFC 7638, all members from this member are included except for the "ek" member. The inclusion of this data in the `Recipient_structure` allows context information to be included in the key derivation.
+* UTF8("JOSE-HPKE rcpt"): A fixed ASCII string identifying the context of the structure.
 
-- recipient_extra_info (string): Contains additional context information that the application includes in the key derivation via the HPKE `info` parameter. Mutually known private information, which is defined in {{NIST.SP.800-56Ar3}}, MAY be used in this input parameter. If no additional context is provided, this value MUST be the empty string "".
+* BYTE(255): A separator byte (0xFF) used to delimit fields.
 
-### Deterministic Serialization for HPKE `info`
+* UTF8(next_layer_alg): Identifies the algorithm with which the HPKE-encrypted key MUST be used. Its value MUST
+  match the "enc" (encryption algorithm) header parameter in the JWE protected header. This field provides JSON context information included in the key derivation process and serves two purposes:
 
-JSON texts that are semantically identical can serialize differently (e.g., member order, whitespace), which would lead to divergent `info` values and failed key agreement.
+  1. Ensures that derived key material is cryptographically domain-separated between the JWE HPKE integrated encryption and Key Encryption modes.
+  2. Mitigates downgrade attacks as discussed in {{?RFC9709}}.
+  For these reasons, this field is mandatory.
 
-To produce the HPKE `info` byte string from a `Recipient_structure`, both sides MUST produce the deterministic JSON representation using the JSON Web Key (JWK) Thumbprint serialization rules {{RFC7638}}:
+* BYTE(255): A separator byte (0xFF) used to delimit fields.
 
-1. Construct the `Recipient_structure` JSON object exactly as defined {{recipient_structure}}.
-2. Prepare the JSON structure based on Section 3.3 of RFC 7638.
-3. Use the resulting JSON structure, base64url-encode it and use the octets as the HPKE `info` value.
+* UTF8(recipient_extra_info): Contains additional context information that the application includes in the key
+  derivation via the HPKE `info` parameter. Mutually known private information, which is defined in {{NIST.SP.800-56Ar3}}, MAY be used in this input parameter. If no additional context is provided, this value MUST be the empty string "".
```

Comment:
Mutually known private information is *not* defined in {{NIST.SP.800-56Ar3}}. As mentioned in April https://github.com/ietf-wg-jose/draft-ietf-jose-hpke-encrypt/pull/41#discussion_r2060452940 but subsequently ignored. 

Can we please stop erroneously saying that it is?

Even the COSE-HPKE document removed it https://mailarchive.ietf.org/arch/msg/jose/s2YuuyPEtLooDoy7HuLjkI3jCKU/